spec/kanal/plugins/batteries_bridge/bridges/telegram_bridge_spec.rb

```diff
@@ -33,10 +33,10 @@
 
     # TODO: kanal of version 0.4.1 doesn't have button_pressed condition pack in batteries
     # TODO: Remove condition pack below on Kanal update
-    core.add_condition_pack :button_pressed do
-      add_condition :contains do
+    core.add_condition_pack :button do
+      add_condition :pressed do
```

Comment:
1) We don't need to check conditions here, we need only to check how bridge converts one thing to another. We can use :flow, :any or even default response for this if I'm not mistaken
2) This condition won't work. It does not have `with_argument`

Reply:
Removed condition pack, checking only for conversion from button_pressed to tg_button_pressed now.